wifi-subsys/components/tools/test/test_cbor.c

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022 Mesh4all <mesh4all.org>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief   Non Volatile Storage
+ *
+ * @author  eduazocar <eduardo@turpialdev.com>
+ */
+#include <stdio.h>
+#include <stdint.h>
+#include "esp_log.h"
+#include "unity.h"
+#include "serialization.h"
+
+static char* key_element[] = {"Key test", "Key_1", "Secret_key"};
+size_t len;
+uint8_t cbor_code[20];
+int64_t val_received;
+
+TEST_CASE("Encode the int value = 32", "cbor")
```

Comment:
please be more clear, i cannot understand,  here you comment that the value is 32 but the parameter pass in the function is 34